src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -7,6 +7,9 @@ import _ from 'lodash';
 // @flow-disable-next
 import { v4 as uuidv4 } from 'uuid';
 import sql from './sql';
+import { StorageAdapter } from '../StorageAdapter';
+import type { SchemaType, QueryType, QueryOptions } from '../StorageAdapter';
+import { relativeTimeToDate } from '../Mongo/MongoTransform';
```

Comment:
I think it makes sense to extract the `relativeTimeToDate`, maybe into `/src/Utils.js`? Otherwise this PR makes the Postgres adapter dependent on the MongoDB adapter, which is the opposite of where we are intending to go, which is moving all storage adapters out of the parse server base codebase, each into its own repo.

Reply:
I moved this to Utils, but I couldn't find a common Utils.spec, so I left the tests inside the MongoTransform spec. If you have a better place, let me know.

Reply:
That's fine I guess